R/Rclone/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Rclone"
+version = v"1.53.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/rclone/rclone.git", "30dae752c9258f6aca9521a205696648fde4a98e")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd rclone/
+
+make
+
+# install manually as `make install` doesn't include $exeext
+install -d ${bindir}
+install -t ${bindir} ${GOPATH}/bin/rclone${exeext}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("rclone", :rclone, "bin")
```

Comment:
I don't think specifying the `bin` directory is needed here (if I understand correctly where this is installed)
```suggestion
    ExecutableProduct("rclone", :rclone)
```

Reply:
Thanks for fixing that. This might have got into a weird state because the initial linux build was successful with `make install` but that installed into `usr/bin`. I later had to edit the install step when it failed on win32 so I simplified things then.